subprojects/distributions-integ-tests/src/integTest/groovy/org/gradle/DistributionIntegrationSpec.groovy

```diff
@@ -36,7 +36,7 @@ import static org.hamcrest.MatcherAssert.assertThat
 
 abstract class DistributionIntegrationSpec extends AbstractIntegrationSpec {
 
-    protected static final THIRD_PARTY_LIB_COUNT = 145
+    protected static final THIRD_PARTY_LIB_COUNT = 146
```

Comment:
`configuration-cache-report.jar` should be consumed at build time and *NOT* included in the distribution.